tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -128,6 +128,9 @@ do
 	fi
 done
 
+# Clean up a custom-node-debugger pod if it exists
+kubectl delete pod -n kube-system custom-node-debugger || true
```

Comment:
Hi @BbolroC !

I leaned these days that `--ignore-not-found=true` is a better option over `|| true` because it won't log any message if the resource doesn't exist ;)

Reply:
Ah, while in here, is there a reason for installing the pod in kube-system namespace? Perhaps because it needs higher-privileges?

Reply:
Thanks for the comment. I will use `--ignore-not-found=true`.

There are 2 reasons to deploy it on a namespace `kube-system`:

- I didn't want to create another dedicate namespace which might need `service account`, `cluster role` and `cluster role binding` just in case
- I just wanted to reuse a guaranteed namespace for the cluster administration